css/index.css

```diff
@@ -104,14 +139,88 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: black; 
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+/* This doesn't make sense, should inherit - something missing 
```

Comment:
Good job here, leaving yourself a todo, but knowing when it's time to move onto another task to meet the time-crunch.